### DISCUSSION TITLE
Feature: Attach scalene to a running process

### DISCUSSION THREAD

**Discussion by laundmo** [plasma-umass/scalene#348] [Ideas]
If at all possible, i would love to be able to attach scalene to a running python process. This would help in profiling issues that only rarely occur in log-running tasks, since it would allow attaching once the issue has been noticed.

I understand if this isnt feasible, scalene is already a amazing tool that helps a lot regardless.

**Comment by thearrow** [comment]
Would love this feature, too! Similar to the attach functionality in py-spy - primarily useful to profile web apps running in gunicorn / uvicorn / etc.  (I suppose this would be related to https://github.com/plasma-umass/scalene/issues/35).
